### PR TITLE
SPLAT-912: introduce vsphere 8 cluster configuration for the vsphere 8 lease pool

### PR DESCRIPTION
**Pull request by rvanderp3**
this PR depends on https://github.com/openshift/release/pull/34789 and https://github.com/openshift/ci-tools/pull/3195

### CODE REVIEW

File: ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh [hunk 1]
```diff
@@ -33,6 +34,19 @@ if [ $((${LEASED_RESOURCE//[!0-9]/})) -ge 88 ]; then
...
 
+# For leases >= 200, run on the IBM Cloud vSphere 8 env
+if [ ${LEASE_NUMBER} -ge 200 ]; then
+  echo Scheduling job on IBM Cloud instance
+  VCENTER_AUTH_PATH=/var/run/vsphere8-secrets/secrets.sh
+  vsphere_url="vcenter.ibmc.devcluster.openshift.com"
+  vsphere_datacenter="IBMCdatacenter"
+  cloud_where_run="IBM8"
+  dns_server="10.177.56.78"
+  vsphere_resource_pool="/IBMCdatacenter/host/IBMCcluster/Resources"
```
[the remark this answers is not in the document]
Comment (jcpowermac) [reply ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh:45]: oh its already in 8, `ipi-ci-clusters`

[the remark this answers is not in the document]
Comment (rvanderp3) [reply ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh:45]: good call, yeah, i created the pool yesterday.  I'll update the path in the PR.

**Pull request merged**